docs/Theming.md

```diff
@@ -9,28 +9,25 @@ Whether you need to adjust a CSS rule for a single component, or change the colo
 
 ## Overriding A Component Style
 
-Every react-admin component provides a `className` property, which is always applied to the root element.
+Every react-admin component inherits an `sx` property from MUI, which is always applied to the root element.
```

Comment:
```suggestion
Every react-admin component exposes the `sx` property from MUI, which is always applied to the root element.
```

Reply:
The `className` information is still valid. I wouldn't replace one over the other, but add the `sx` prop to it